Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -254,7 +254,7 @@ extension BrowserViewController: WKNavigationDelegate {
       
       let domainForMainFrame = Domain.getOrCreate(forUrl: mainDocumentURL, persistent: !isPrivateBrowsing)
       // Enable safe browsing (frodulent website warnings)
-      webView.configuration.preferences.isFraudulentWebsiteWarningEnabled = domainForMainFrame.isShieldExpected(.SafeBrowsing, considerAllShieldsOption: true)
+      webView.configuration.preferences.isFraudulentWebsiteWarningEnabled = Preferences.Shields.googleSafeBrowsing.value
```

Comment:
I don't think this is needed anymore, the preference is set when making the web view, and I don't think this will be handled without resetting the configuration/remaking the web view? We may want to test if this works